classes/form/password_form.php

```diff
@@ -57,5 +57,10 @@ public function definition() {
             util::lang_string('form_userpass_forgot_class'),
             new moodle_url('/local/signin/forgot.php'),
             util::lang_string('form_password_forgot_label')));
+
+        $mform->addElement('html', sprintf(util::ELEMENT_WRAPPER,
+            util::lang_string('form_userpass_forgot_class'),
```

Comment:
This looks like it has been copied and not changed

Reply:
Yeah its been copied from the above line in the same page. However its been changed as per the requirement for having a link in Signin page.
I have changed the url which will redirect to the Unlock Account page.
I have changed the lang_string key and defined in lang file to show "My account is unlocked" in Signin page.
Element Wrapper is not changed as it is have to be in same class as the others link is in like forgot passord/forgot username.

Reply:
ah ok